package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "symphony-react-autosuggest",
-  "version": "3.7.3-symphony.3",
+  "version": "3.7.3-symphony.5",
```

Comment:
PR 4 (SEARCH-409) is still under progress, please make this rev 4 and good to go.

Reply:
I'll just close this PR and add this change as part of that PR.